_boards_cm/waveshare-dual-gigabit-ethernet-base-board.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: board
+title: "Waveshare's Dual Gigabit Ethernet Base Board for the CM4"
+short_description: A CM4 base board with a large number of connection options.
+status: production
+picture: "/images/board-waveshare-dual-gigabit-ethernet.jpg"
```

Comment:
Note: The image itself needs renaming to `board-waveshare-dual-gigabit-ethernet.jpg`.

Reply:
Ah, I thought I might have had that happen to one ofthem last night, but I couldn't figure out which one. Thanks for catching it.